Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -46,6 +47,7 @@ public IHidServer(ServiceCtx context) : base(context.Device.System.HidServer)
             _npadJoyAssignmentMode      = HidNpadJoyAssignmentMode.Dual;
             _npadHandheldActivationMode = HidNpadHandheldActivationMode.Dual;
             _gyroscopeZeroDriftMode     = HidGyroscopeZeroDriftMode.Standard;
+            _isFirmwareUpdateAvailableForSixAxisSensor = false;
```

Comment:
Degroup from previous assignation or align with them

```suggestion

            _isFirmwareUpdateAvailableForSixAxisSensor = false;
```